### DISCUSSION TITLE
Amazing Work!

### DISCUSSION THREAD

**Discussion by wojciech-kulik** [doronz88/pymobiledevice3#851] [Show and tell]
@doronz88  I would like to thank you for your amazing work and the quick support!

I've just finished integration between this tool and my plugin. Works great. It made it possible to work with apps on devices with iOS below 17 + enabled debugging on all physical devices.

Here you can see the results of the integration:
https://github.com/wojciech-kulik/xcodebuild.nvim?tab=readme-ov-file#availability-of-features

And if you are curious about the code, here is the main integration:
https://github.com/wojciech-kulik/xcodebuild.nvim/blob/main/lua/xcodebuild/platform/device_proxy.lua

Thank you once again for this great piece of software! 🍻

**Comment by doronz88** [comment]
Thanks! and a really cool project :)
You could technically add iOS 17 device support if you also make sure to run a `sudo pymobiledevice3 remote tunneld` in the background

**Reply by wojciech-kulik** [reply]
I did it :)) I detect iOS version and for iOS 17+ I start secure tunnel :) However, it’s problematic to run sudo from Neovim, so I wrapped this command with a small script, so that the user can set passwordless sudo permission to it :)
